custom-logger/README.md

```diff
@@ -0,0 +1,187 @@
+# Customer Logger demo
+
+This project demonstrates how to capture log entries produced by Workers, Workflows and Activities, and process them
+using a third party logging libraries (in this case, the [Winston](https://github.com/winstonjs/winston) package).
+
+The [Logging docs](https://docs.temporal.io/dev-guide/typescript/observability#logging) explain some of the code in this sample.
+
+### Running this sample
+
+1. `temporal server start-dev` to start [Temporal Server](https://github.com/temporalio/cli/#installation).
+1. `npm install` to install dependencies.
+1. `npm run start.watch` to start the Worker.
+1. In another shell, `npm run workflow` to run the Workflow.
+
+<details>
+<summary>
+Sample worker output
+</summary>
+
+```
+2023-09-05T18:19:39.646Z [undefined] info: Creating worker {
```

Comment:
undefined?

Reply:
Good catch. The original sample was adding a `label` metadata (`worker`, `workflow` or `activity`). That required registering different loggers on each context, which I definitely want to avoid.

For the moment, I fixed this with an heuristic in the logger adaptor ([see here](https://github.com/temporalio/samples-typescript/pull/279/files#diff-ddf423b693396f737a7ca3e8f5e525a9fb5e213493dd1530bfa853bcbfa31abfR18)), just so that we can move on to merging this PR, but I can easily imagine cases where that would guess incorrectly.

I think we should provide something similar to that 'label' property out of the box.